templates/main.php

```diff
@@ -24,18 +24,18 @@
                 </div>
                 <div class="lot__info">
                     <span class="lot__category"><?= e($unit['category']); ?></span>
-                    <h3 class="lot__title"><a class="text-link" href="pages/lot.html"><?= e(
-                                $unit['name']
-                            ); ?></a>
+                    <h3 class="lot__title"><a class="text-link" href="pages/lot.html"><?= e($unit['name']); ?></a>
                     </h3>
                     <div class="lot__state">
                         <div class="lot__rate">
                             <span class="lot__amount">Стартовая цена</span>
                             <span class="lot__cost"><?= e(format_amount($unit['price'])); ?></span>
-                            <!--<b class="rub">р</b>-->
                         </div>
-                        <div class="lot__timer timer">
-                            12:23
+                        <div class="lot__timer timer <?= e(
+                            countdown($unit['fin_time'])[0]
+                        ) === '00' ? 'timer--finishing' : '' ?>">
+                            <?= e(countdown($unit['fin_time'])[0]); ?>:
```

Comment:
this implementation calls countdown function couple time. why? first call can give you both hours and minute. So I recommend to run it once, save result to variable and use it. 

This way makes execution simpler. Yes, here it doesn't cost too much, countdown function is simple one. But it's always better to avoid non required calculations and any long going operations like work with db.

```
$countdown = countdown($unit['fin_time']);
<?= e($countdown[0]) ?>
<?= e($countdown[1]) ?>
```